src/graalvm.ts

```diff
@@ -23,13 +24,27 @@ const GRAALVM_TAG_PREFIX = 'vm-'
 // Support for GraalVM for JDK 17 and later
 
 export async function setUpGraalVMJDK(
-  javaVersionOrDev: string
+  javaVersionOrDev: string,
+  gdsToken: string
 ): Promise<string> {
   if (javaVersionOrDev === c.VERSION_DEV) {
     return setUpGraalVMJDKDevBuild()
   }
+  const isTokenProvided = gdsToken.length > 0
   let javaVersion = javaVersionOrDev
   const toolName = determineToolName(javaVersion, false)
+  if (javaVersionOrDev === '17' && !isTokenProvided) {
+    core.warning(
+      'This build uses the last update of Oracle GraalVM for JDK 17 under the GFTC. More details: https://github.com/marketplace/actions/github-action-for-graalvm#notes-on-oracle-graalvm-for-jdk-17.'
```

Comment:
This warning will be displayed for users using `java-version: '17'`. The link should resolve to this section in the README: https://github.com/graalvm/setup-graalvm/pull/109/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R194